### PR TITLE
Add version to tabulate CMake

### LINKED ISSUE #51: It's not possible to import version of tabulate on cmake
The command:

`message(STATUS "${tabulate_VERSION}")`

Returns empty.

I can fix it in a PR!

### PR DESCRIPTION
**Pull request by marimeireles**
Fixes #51

### CODE REVIEW

File: include/tabulate/tabulate.hpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//             #define TABULATE_API __declspec(dllimport)
+//         #endif
+//     #endif
+// #else
+//     #define TABULATE_API
+// #endif
+
+// Project version
+#define TABULATE_VERSION_MAJOR 0
+#define TABULATE_VERSION_MINOR 1
```
Comment (p-ranav) [inline include/tabulate/tabulate.hpp:54]: Wouldn't it be major version 1 and minor version 3?

**Review by p-ranav — CHANGES_REQUESTED** [review]

**Comment by p-ranav** [comment]
Can you also update the single_include version? Thanks!!

File: include/tabulate/tabulate.hpp (hunk 1 above)
Comment (marimeireles) [reply include/tabulate/tabulate.hpp:54]: oh yeah! thanks

**Review by p-ranav — APPROVED** [review]

**Comment by p-ranav** [comment]
Thank you!

**Pull request merged**